constants_unix.go

```diff
@@ -1,4 +1,6 @@
-// +build darwin dragonfly freebsd netbsd openbsd solaris
+// +build darwin dragonfly freebsd netbsd openbsd
+// +build !solaris
+// +build !illumos
```

Comment:
Will `constants_solaris.go` automatically be included on illumos?

Reply:
Yes illumos is a descendant of solaris in the go arch system. Not historically accurate but it prevents us from having to fix a huge amount of packages.